tests/zeek/file-analyzer-nested.zeek

```diff
@@ -24,6 +25,11 @@ event text::data3(f: fa_file, data: string)
 	print "data3", f$id, data;
 	}
 
+event Spicy::max_file_depth_exceeded(f: fa_file, args: Files::AnalyzerArgs, limit: count)
+	{
+	print "max file depth exceeded", f$id, args, limit;
```

Comment:
This should be showing up `output-max`, but the `grep` is filtering it out, so that we don't have it the baseline; probably want to use some other text here not matching the grep.

Also, would be good check `notice.log` for the new `NOTICE`.